scanpy_scripts/cmds.py

```diff
@@ -210,3 +214,31 @@
     arg_desc=_IP_DESC,
     opt_set='plot_paga'
 )
+
+COMBAT_CMD = make_subcmd(
+    'combat',
+    combat,
+    cmd_desc='ComBat function for batch effect correction',
+    arg_desc=_IO_DESC
+)
+
+HARMONY_INTEGRATE_CMD = make_subcmd(
+    'harmony_integrate',
+    sce.pp.harmony_integrate,
+    cmd_desc='Use harmonypy [Korunsky19] to integrate different experiments.',    
+    arg_desc=_IO_DESC,
+)
+
+BBKNN_CMD = make_subcmd(
+    'bbknn',
+    bbknn,
+    cmd_desc='Batch balanced kNN [Polanski19].',
+    arg_desc=_IO_DESC,
+)
+
+MNN_CORRECT_CMD = make_subcmd(
+    'mnn_correct',
```

Comment:
Similarly, can "mnn_correct" be shortened to "mnn"? Just for lazy command line users...